### DISCUSSION TITLE
Welcome to obsidian-textgenerator-plugin Discussions!

### DISCUSSION THREAD

**Discussion by nhaouari** [nhaouari/obsidian-textgenerator-plugin#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gncorbin** [comment]
Hi All,
I'm enjoying this plugin and using it daily to write emails, revise paragraphs, think through an outline of an issue/topic. Really appreciate the quick work you're doing on this!

**Comment by ricraftis** [comment]
The link to the Discord server appears to have expired. Can it be set to Never please?

**Comment by connect-m** [comment]
The Text Generator PlugIn in Obsidian uses v0.5.13 but here version 0.5.9. is available. The update procedure of Obsidian plugIns does not update it though. How can I update it anyway?
